src/command_transfer.h

```diff
@@ -0,0 +1,63 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2015 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include "usb/usb-device.h"
+
+#include <vector>
+#include <stdint.h>
+
+namespace librealsense
+{
+    namespace platform
+    {
+        class command_transfer
+        {
+        public:
+            virtual std::vector<uint8_t> send_receive(
+                const std::vector<uint8_t>& data,
+                int timeout_ms = 5000,
+                bool require_response = true) = 0;
+
+            virtual ~command_transfer() = default;
+        };
+
+        class command_transfer_usb : public command_transfer
```

Comment:
Check whether the base class's dtor should be made virtual

Reply:
it is virtual